status-im/status-react
# Mailserver issue fixed in status-go

fixes #...

### Summary

`status-go` got fix for [issue about mailserver disconnects](https://github.com/status-im/status-go/issues/2115). It shouldn't affect `status-react` but worth checking


### Testing notes
`eth.test` fleet updated with mailserver created from fixed `status-go`

#### Platforms
- Android
- iOS

#### Areas that maybe impacted
mailserver connection


### Steps to test
- run `status-react`
- switch to eth.test fleet and open some chat
- make sure there is no disconnects from mailserver every 2 minutes


status: ready
